### ISSUE TITLE
Check and update all rules to consider nested props

### ISSUE THREAD

**Issue by dryoma** [kristerkari/stylelint-scss#62]
Like in
```scss
.funky {
  font: 20px/#{$font-size*1.2} fantasy { // operator-no-unspaced should warn
    weight: bold;
  }
}
```

**Comment by dryoma** [comment]
Possibly won't be needed to implement on the linter's side if [this](https://github.com/postcss/postcss-scss/issues/39) gets resolved.

**Comment by kristerkari** [comment]
@dryoma What's the status of this? Should we go ahead and check for nested props inside the rules? 

I guess we still haven't decided about the forking of `postcss-scss`:
https://github.com/kristerkari/stylelint-scss/issues/74

We could maybe postpone this issue and release 1.4.0 as we have had some things pending for a while:
https://github.com/kristerkari/stylelint-scss/blob/master/CHANGELOG.md#head

**Comment by dryoma** [comment]
Sorry, I decided to not proceeding with this until we decide on postcss-scss.

So yes, totally fine with me to release with the current progress.
